### PR TITLE
Port to Python 3

### PR DESCRIPTION
**Pull request by Bottersnike**
Tested within `sugar-activity3`. When testing within a local sugar environment, I encountered a DBus error where it was attempting to cast the repr of a DBus into an int. I believe this to be an issue with my local sugar environment and not with this activity, however this needs tested by someone with a confirmed-working local sugar environment.

### CODE REVIEW

File: genxo.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python5
```
Comment (tchx84) [inline genxo.py]: Python 5?

**Review by tchx84 — CHANGES_REQUESTED** [review]
Just a silly typo.

**Comment by tchx84** [comment]
Other comments fro our IRC chat:

* the color of the turtle seems changed, but it might be an earlier regression, extra kudos if you fix it ;)
* make sure to double check with some of the examples from https://help.sugarlabs.org/en/turtle_in_a_pond.html to import custom strategies

Thanks! this is awesome.

**Comment by Bottersnike** [comment]
All builtin strategies appear to work, and all strategies on the wiki appear to be functioning python 3 code. I'm unable to test importing from the journal as my local sugar instance refuses to start turtlepond (due to unrelated issues) however manually loaded a number of custom strategies by editing the source and none failed to work

File: genxo.py (hunk 1 above)
Comment (Bottersnike) [reply genxo.py]: Fixed by the below squashed push

**Comment by quozl** [comment]
Thanks.  Reviewed.  Tested starting from Sugar.

**Pull request merged**